symfony/assets/scripts/entry/general.ts

```diff
@@ -32,3 +33,43 @@ jQuery(() => {
     config.save();
   });
 });
+
+jQuery(() => {
+  const newsNavlink = jQuery("#navlink-news");
```

Comment:
I use a convention of prepending any jQuery objects with `$`. Do you agree with this, or would you rather not? NOTE: this is not a convention coming from PHP variables.

Reply:
More than anything, you're definitely the lead for the codebase, so if you'd prefer any particular conventions I am happy to adopt them.

My personal thoughts are that it might not be necessary in TypeScript. In pure JavaScript, I think there's a lot of value — what functions are available are totally different if it's a vanilla DOM node vs. jQuery object. But in TypeScript, the static compiler and the IDE will tell you if it's jQuery or not, so the Hungarian-notation doesn't offer quite the same value. Without a linter to keep it accurate, it could also become inaccurate over time (we refactor `$myDiv` to no longer be a jQuery object but forget to update the name).

Reply:
Fair point. I started creating a document explaining code practices/naming/deprecations and this may become a new entry there.